python/lsst/sims/catUtils/utils/LightCurveGenerator.py

```diff
@@ -309,11 +314,14 @@ def get_pointings(self, ra, dec, bandpass=('u', 'g', 'r', 'i', 'z', 'y'), expMJD
 
         return obs_groups_out
 
-    def _get_query_from_group(self, grp, chunk_size):
+    def _get_query_from_group(self, grp, chunk_size, lc_per_field=None):
         """
```

Comment:
For consistency I think we should properly document all input and output parameters here as we have elsewhere.